setup_with_snap.md

```diff
@@ -43,20 +43,20 @@ lorawan_bridge:
 - Every variable has to be set.
 
 ```bash
-  sudo snap set wisevisionproj config-file=<path_to_zenoh_config_file>
-  sudo snap set wisevisionproj ros-config-file=<path_to_ros_config_parameters_file>
-  sudo snap set wisevisionproj email-username-notification=<email_username_to_send_notications>
-  sudo snap set wisevisionproj email-password-notification=<email_password_to_send_notications>
-  sudo snap set wisevisionproj email-recipients-notification=<email_recipients_to_send_notications>
-  sudo snap set wisevisionproj chirpstack-api-key=<chirpstack_api_key>
-  sudo snap set wisevisionproj backend-api-url=<backend_api_url>
+  sudo snap set wisevision-iot config-file=<path_to_zenoh_config_file>
+  sudo snap set wisevision-iot ros-config-file=<path_to_ros_config_parameters_file>
+  sudo snap set wisevision-iot email-username-notification=<email_username_to_send_notications>
+  sudo snap set wisevision-iot email-password-notification=<email_password_to_send_notications>
+  sudo snap set wisevision-iot email-recipients-notification=<email_recipients_to_send_notications>
+  sudo snap set wisevision-iot chirpstack-api-key=<chirpstack_api_key>
+  sudo snap set wisevision-iot backend-api-url=<backend_api_url>
```

Comment:
Same as with not if anything is optinoal add note. also im wondering if it's beneficial to run a script or something to set those variables based on the .env file.